common/cmake/devices.cmake

```diff
@@ -547,8 +556,12 @@ function(DEFINE_DEVICE)
     RR_PATCH_TOOL ${DEFINE_DEVICE_ARCH} RR_PATCH_TOOL
   )
   get_target_property_required(RR_PATCH_CMD ${DEFINE_DEVICE_ARCH} RR_PATCH_CMD)
-  get_target_property_required(ROUTE_CHAN_WIDTH ${DEFINE_DEVICE_ARCH}
-    ROUTE_CHAN_WIDTH)
+
+  if("${DEFINE_DEVICE_ROUTE_CHAN_WIDTH}" STREQUAL "")
+    get_target_property_required(ROUTE_CHAN_WIDTH ${DEFINE_DEVICE_ARCH} ROUTE_CHAN_WIDTH)
```

Comment:
Is this `if` statement here twice?  See line 602?

Reply:
I've integrated those two conditions into a single one.

Reply:
The lines are 602 are still there?  Am I missing something?